sdk/purview/purview-datamap-rest/test/public/glossary.spec.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Recorder } from "@azure-tools/test-recorder";
+import { createRecorder } from "./utils/recordedClient";
+import { assert } from "chai";
+import { Context } from "mocha";
+import { createClient } from "./utils/recordedClient";
+
+
+
+describe("purview datamap glossary test", () => {
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("Should create a glossary", async () => {
+	const client = await createClient(recorder);
```

Comment:
personally i would prefer letting client creation in beforeEach and this could benefit cross all cases. But considering there is only one case i am okay with this.